conans/test/command/package_test.py

```diff
@@ -15,30 +15,43 @@ def package_with_destination_test(self):
         client = TestClient()
 
         def prepare_for_package(the_client):
+            the_client.all_output = ""
             the_client.save({"src/header.h": "contents"}, clean_first=True)
             the_client.run("new lib/1.0 -s")
 
-            # don't need real build
+            # don't need build method
             tools.replace_in_file(os.path.join(client.current_folder,
-                                               "conanfile.py"), "cmake =",
-                                  "return\n#")
+                                               "conanfile.py"),
+                                               "def build(self):",
+                                               "#def build(self):")
+            tools.replace_in_file(os.path.join(client.current_folder,
```

Comment:
Why all these replaces if you are not building?

Reply:
You have to comment the whole `build()` so as to not get the `package(): WARN: No files copied!`.
In packages without build mehtod (header only) the warning is skipped because there are no generated binaries and nothing to copy from there -> Warning should not appear

Reply:
replace ``build(`` => ``build_skip(``

The less code, the better